__tests__/TestHelper.js

```diff
@@ -364,6 +364,53 @@ const TestHelper = {
 			cosignatures,
 			innerTransactions
 		};
+	},
+	generateNodePeerStatus: isAvailable => {
```

Comment:
there are inline tabs here and in other places, is lint not running properly?

Reply:
> there are inline tabs here and in other places

what do you mean by inline tabs?

>  is lint not running properly?

I didn't get any error when running `npm run lint: fix`

Reply:
are you sure lint is configured correctly on test code?
notice it is `return<TAB>{`

Reply:
> are you sure lint is configured correctly on the test code?

yes, but not sure why the lint fix does not detect this issue on `return<TAB>{`.